WooCommerce/Classes/Extensions/UIImage+Woo.swift

```diff
@@ -1218,8 +1218,8 @@ extension UIImage {
         return UIImage(named: "woo-waiting-customers")!
     }
 
-    static var puzzleExtensionsImage: UIImage {
-        return UIImage(named: "woo-puzzle-extensions")!
+    static var puzzleImage: UIImage {
+        return UIImage(named: "puzzle")!
```

Comment:
Same as the previous updates, I'm updating this illustration to use the same name we have in the Figma library.